src/Message/Message.tsx

```diff
@@ -4,6 +4,7 @@ import styled, { css } from 'styled-components'
 import { Box } from '../Box'
 import { Icon } from '../Icon'
 import { theme } from '../theme'
+import { useDeprecatedWarning } from '../utils/deprecated'
 
```

Comment:
Could we have the `@deprecated` tag on the `Message` component as well so it appear as such in VS code?